### PR TITLE
Disable ceph packages check

### PR DESCRIPTION
**Pull request by rakeshgm**

### CODE REVIEW

File: teuthology/run.py [hunk 1]
```diff
@@ -249,6 +249,8 @@ def get_initial_tasks(lock, config, machine_type):
             {'internal.setup_additional_repo': None},
             {'kernel.install_latest_rh_kernel': None}
         ])
+        # removing check_packages from init_tasks for now
+        init_tasks.remove({'internal.check_packages': None})
```
Comment (kshtsk) [inline teuthology/run.py]: 1) Why is this required? 
2) Why just not remove the internal.check_packages at the point where it is added at line 177?

Comment (rakeshgm) [reply teuthology/run.py]: we are skipping this only rhel as we dont have have EL8 packages available.

Comment (vasukulkarni) [reply teuthology/run.py]: @kshtsk FYI: this is for 'rh' branch not master, It will affect only downstream runs(not upstream in sepia lab)

Comment (kshtsk) [reply teuthology/run.py]: @vasukulkarni thanks :-) I was just wondering, sorry for inconvenience
